### ISSUE TITLE
Problem on content types with Draft/Publish feature enabled

### ISSUE THREAD

**Issue by dappiu** [Cringe-Studio/strapi-auto-uuid#3]

**Comment by scerelli** [comment]
thanks @dappiu i will check this out

**Comment by scerelli** [comment]
Hello @dappiu i tried to follow your steps, but i could correctly save the new custom type with only `UUID` field in it. Do you have any further deatils to share? a loom or something

**Comment by jonrot1906** [comment]
I actually found that the described errors here and in #4, including the `The value does not match the regex` error, are occurring, when trying to populate existing records with a UUID. When creating a new record, everything is working fine and the UUID is also appearing in the API data.

**Comment by scerelli** [comment]
@jonrot1906, so if I understand correctly, I create a new model without auto UUID. Then, I create a few records, and after that, I add auto UUID to the model. It shouldn't work, right?

**Comment by jonrot1906** [comment]
> @jonrot1906, so if I understand correctly, I create a new model without auto UUID. Then, I create a few records, and after that, I add auto UUID to the model. It shouldn't work, right?

Exactly. But if you create the records after you added the auto UUID, everything is fine.
